infrastructure/modules/ecs/variable.tf

```diff
@@ -60,4 +60,5 @@ variable "alarm_actions_arn_list" {
 
 locals {
   is_sandbox = contains(["ndra", "ndrb", "ndrc", "ndrd"], terraform.workspace)
+  is_production = contains(["prod", "pre-prod", "production"], terraform.workspace)
```

Comment:
The ticket mention 'This only needs to be enabled for pre-prod' was there another requirement somewhere else?

Reply:
I think the adding stuff for prod was in a verbal discussion about this, but I can remove these for now just for the sake of the ticket add those in later once we're spinning up a production

Reply:
Done

Reply:
This will be required on pre-prod plus environments, even though they don't technically exist yet, I'm happy with them being included here as it saves us a job down the line.

Good to call out though :)

Reply:
As per Scott's comment I've put these back in